bin/nodenv-each

```diff
@@ -2,32 +2,50 @@
 #
 # Summary: Execute a command for each Node version
 #
-# Usage: nodenv each [-v] <command> [arg1 arg2...]
+# Usage: nodenv each [-sv] <command> [arg1 arg2...]
 #
 # Executes a command for each Node version by setting NODENV_VERSION.
 # Failures are collected and reported at the end.
 #
+#    -s  Include the system `node`.
 #    -v  Verbose mode. Prints a header for each node.
 #
 set -e
 [ -n "$NODENV_DEBUG" ] && set -x
 
 # Provide nodenv completions
-case "$1" in
-  --complete )
-    echo --verbose
-    nodenv shims --short
-    exit
-    ;;
-  -v | --verbose )
-    verbose=1
-    shift
-    ;;
-  ""  | -* )
-    nodenv-help --usage each >&2
-    exit 1
-    ;;
-esac
+for arg in "$@"; do
+  case "$arg" in
+    --complete )
+      echo --verbose
+      echo --system
+      nodenv shims --short
+      exit
+      ;;
+    -v | --verbose )
+      verbose=1
+      shift
+      ;;
+    -s | --system )
+      include_system=1
+      shift
+      ;;
+    -sv | -vs )
```

Comment:
Neither nodenv itself, nor any of the other nodenv plugins support combining short flags, so I'd rather not add it here. The lack of consistency with the rest of nodenv would either cause it to not be used (as its support would be unique to nodenv-each) or lead to unsupported use with other nodenv commands (as people may attempt to combine shortflags with other non-supporting nodenv commands).